Dockerfile.machine-config-operator

```diff
@@ -1,20 +1,18 @@
+# This is the "all in one" image for the MCO; we previously
```

Comment:
I know it's a pain but `Dockerfile` must be updated as well till we remove this

Reply:
Oh right I changed the wrong one, argh.